frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -403,7 +404,7 @@ class Chart extends Component<Props, State> {
     const valueTo = this.props.data.chartTotal;
     const valueDiff = valueTo ? valueTo - valueFrom : 0;
     this.setState({
-      difference: ((valueDiff / valueFrom) * 100),
+      difference: ((valueDiff / valueFrom)),
```

Comment:
with last approach we calculated % number, but Intl.NumberFormat expects values 0-1 for 0% - 100%.

Intl.NumberFormate is used for localizePercentage in utils/localize.ts.

```js
new Intl.NumberFormat(locale, {
      maximumFractionDigits: 2,
      minimumFractionDigits: 2,
      signDisplay: 'auto',
      style: 'percent',
    });
```